torch/csrc/distributed/rpc/rref_proto.h

```diff
@@ -15,24 +15,28 @@ namespace rpc {
 // TODO: Remove all these messages and use rpc + registered functions instead.
 class TORCH_API RRefMessageBase : public RpcCommandBase {
  public:
-  RRefMessageBase(const RRefId& rrefId, MessageType type)
-      : rrefId_(rrefId), type_(type) {}
+  RRefMessageBase(const RRefId& rrefId, MessageType type, DeviceMap&& deviceMap)
```

Comment:
not 100% sure, but looks like only Script/PythonRRefFetchCall needs this device map. Does it make sense to only add this device map to those two messages?

Reply:
agree